TheFourFifths/consus-client
# Client-side API and Store code for checkout flow

### Summary

Setup the client-side API calls and stores to for the data once the calls complete

### Checklist


Finally, is it ready to be reviewed and merged: yes :white_check_mark:

### How to Review

1. Ensure any needed stores for checkout exist
2. Ensure any api functions needed for checkout exist.
3. Ensure tests are useful and make sense.

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-57](https://msoese.atlassian.net/browse/TFF-57)
- [TFF-69](https://msoese.atlassian.net/browse/TFF-69)
